topics:
  input:
    bounding_boxes: "/darknet_ros/bounding_boxes"
  output:
    pose_estimate: "/estimate/dnn_cv_pose"
